src/components/questionStructure/Task/variants/HandwritingTask/HandwritingEditor.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Card, Flex } from '@radix-ui/themes'
+import { MathJax } from 'better-react-mathjax'
+import React from 'react'
+
+import Markdown from '../../../../Markdown'
+import Canvas from './Canvas'
+
+const HandwritingEditor: React.FC<{
+  username: string
+  latex: string
+  onAnswerChange: (value: string) => void
+  questionText: string
+}> = ({ username, latex, onAnswerChange, questionText }) => {
+  return (
+    <Flex direction="column" flexGrow="1" align="center" gap="3">
+      <Markdown>{questionText}</Markdown>
```

Comment:
```suggestion
```

Reply:
See https://github.com/the-answerbook-project/answerbook-ui/pull/20/files#r1684254378